scribble-doc/scribblings/scribble/acmart.scrbl

```diff
@@ -74,6 +74,47 @@ number of options may be used:
 }|
 
 If multiple font size options are used, all but the last are ignored.
+
+The ACM documentation (version 1.54, 2018-07-16, by Boris
```

Comment:
maybe use a `@margin-note{}` instead of parentheses?

Reply:
Oh my... in fact, the "margin-note" function (when used in that file, at least) produces extraordinarily prominent text. I don't think that's an improvement. Here's what it looks like...
<img width="503" alt="screen shot 2018-08-28 at 14 04 16" src="https://user-images.githubusercontent.com/226617/44741516-4fe08f80-aacb-11e8-90ec-8d557b4c0789.png">